lib/README.md

```diff
@@ -0,0 +1,85 @@
+<!-- THIS IS AUTOGENERATED FILE. DO NOT CHANGE IT MANUALLY! -->
```

Comment:
How is this file generated? Will it be correctly auto generated after being moved to `lib/`?

Reply:
cc @denisshevchenko